app/assets/stylesheets/overrides/_eui.scss

```diff
@@ -14,3 +14,7 @@
     color: #963b3b;
   }
 }
+
```

Comment:
The eui 1.1.7 change changed our formatting and this override returns us to what it used to look like.